src/wp-includes/class-wp-scripts.php

```diff
@@ -289,6 +289,24 @@ public function do_item( $handle, $group = false ) {
 		$cond_after  = '';
 		$conditional = isset( $obj->extra['conditional'] ) ? $obj->extra['conditional'] : '';
 
+		// Add extra attributes based on the strategy.
+		$extra_atts = '';
+		$strategy   = $this->get_strategy( $handle );
+		switch ( $strategy ) {
```

Comment:
add a blocking or `beforeInteractive` strategy that ensues the script is in the header and no async/defer is applied.